docs/concepts.rst

```diff
@@ -1,119 +1,108 @@
 
 Project Structure
 -----------------
 
-A CumulusCI project represents an application or a major component of an application on the Salesforce platform. Each project corresponds with a single managed package, or with an org-based implementation. In this documentation, we'll often speak about a package; if you're building a project that isn't packaged, please take this to refer to an unmanaged package within which your implementation is deployed.
+Keychain
+^^^^^^^^
 
-Together with the core package, a CumulusCI project encompasses a variety of unpackaged application components, automation, tests, and other support material. We'll quickly survey the primary components of a CumulusCI project here. Each component is covered in more depth later in this guide.
+Each project comes with a Salesforce CLI keychain, a unique identifier of characters to separate a project and its files from any other. This deters collisions with another org. (Example: Most projects contain a "dev" org. While the name is ubiquitous across Salesforce orgs, the keychain attached is what makes it unique.) 
 
-Automation Configuration (``cumulusci.yml``)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Project Directory
+^^^^^^^^^^^^^^^^^
 
-``cumulusci.yml`` is the definition of the project's automation. It includes information about the project itself as well as customizations that define how scratch orgs are built and provide other important functionality to be used as part of the project's lifecycle.
+The root directory of your CumulusCI project. Because each project is linked to a single GitHub repository, you can always be confident of the scope of the actions you're taking. And thanks to the keychain, CumulusCI knows which project you are working on by the current working directory of your shell. 
 
-When you work with CumulusCI in a project, the contents of your ``cumulusci.yml`` are merged with the ``cumulusci.yml`` that comes with CumulusCI and defines out-of-the-box functionality. From time to time we'll refer to that as the standard library. Learn more about customizing CumulusCI automation in TODO: link Customization section.
+[NOTE: Save yourself a headache by making sure you're in the correct repository for your project before running project-specific commands; otherwise, your project will produce an error. (This tends to be one of the biggest troubleshooting issues that developers run into with CumulusCI. Check for this first and you might save yourself an extra trip to this guide.)]
```

Comment:
Helpful hint? Annotation?

Reply:
Dang, I didn't know we had all of those options :)